ncsa_storage/yaml-builder.py

```diff
@@ -35,7 +35,7 @@
 import sys
 import yaml
 
-def _build_yaml(data_path, pvc_name, hostname, output_dir, template_dir):
+def _build_yaml(data_path, pvc_name, hostname, instance, output_dir, template_dir):
```

Comment:
I wonder if this file could be modified to create a specified number of instances instead of needing to take an instance parameter with 20 almost-identical input files?

Reply:
This is do-able. However, current technique is less elegant but looks more flexible to me. So I keep in mind your proposal and might implement it when the pv/pvc creation process will be stabilized. Please note that it may be possible to do it with some k8s ecosystem tools (like a future kustomize version maybe), and that all of this is generated automatically on k8s cloud with dynamic volume provisionning enabled (like GKE)...